### DISCUSSION TITLE
Add version to PR description

### DISCUSSION THREAD

**Discussion by laurentsimon** [renovatebot/renovate#16818] [Suggest an Idea]
When a GHA is used as `uses: the/action@v3`, the PR description from renovatebot looks like:

`abcd -> efgch`.

Would it be possible to indicate the version in the PR description? For example, something like:

`(v3) abcd -> efgch`

or, in cases where a v3.x.y corresponds to the v3 tag:

`v3.xy -> v3.x.w`

**Comment by PhilipAbed** [comment]
i have never seen renovate write versions in the title, unless its a major update maybe 
maybe in the branch name
but you can see the version inside the PR body for sure

why do you need that?

**Reply by rarkins** [reply]
They want visibility into which tag's SHA is being updated without needing to look into the files diff

**Comment by rarkins** [comment]
No to the v3.x.y part - it's too complex to implement

**Comment by rarkins** [comment]
This should be possible using our commitMessage construct, because the PR title reuses it. So Eg commitMessageTopic or commitMessageExtra could be changed to accommodate it
